Bottom: de09e2c6782ad049e0cd2b8b1a2b9f66343f8367
Top:    caee3e13e3c18d681304315aa3eb533c41e7c75f
Author: Ian Kent <ikent@redhat.com>
Date:   2013-10-02 08:52:59 +0800

Refresh of autofs-5.0.7-samples-auto_smb-add-logic-to-obtain-credentials.patch

---

diff --git a/samples/auto.smb b/samples/auto.smb
index 6975263..3ec2e90 100755
--- a/samples/auto.smb
+++ b/samples/auto.smb
@@ -23,7 +23,7 @@
 
 get_krb5_cache() {
     cache=
-    uid=$(id -u)
+    uid=${UID}
     for x in $(ls -d /run/user/$uid/krb5cc_* 2>/dev/null); do
         if [ -d "$x" ] && klist -s DIR:"$x"; then
 	    cache=DIR:$x
